### ISSUE TITLE
If you fail to publish, following publish attempts will try to fetch from the wrong branch.

### ISSUE THREAD

**Issue by rawleyfowler** [ocaml-opam/opam-publish#142]
If I try to publish and fail (invalid github key permissions, etc). Following attempts will yield this:
```
Fetching the package repository, this may take a while...
[ERROR] Uncaught exception: "/usr/local/bin/git fetch --multiple origin user"
        exited with code 128 "fatal: no such remote or remote group: user"
```

Obviously it should be trying to fetch from `origin main` and not `origin user` right?

I get this error on FreeBSD, Linux, and MacOS if I fail to publish. I have no idea how to make this go away, so I just publish manually from now on.

**Comment by kit-ty-kate** [comment]
I think this should have been fixed with #140. Could you confirm?

**Comment by rawleyfowler** [comment]
Yup. Looks good. Thanks!
